firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/Serial/Backend/Backend.h

```diff
@@ -53,7 +53,7 @@ static const auto state_sync_schedule = Util::make_array<const StateOutputSchedu
 using BackendMessage = Protocols::Message<
     Application::StateSegment,
     Application::MessageTypeValues,
-    Protocols::DatagramProps<
+    Protocols::CRCElementProps<
```

Comment:
We caught a bug, where the CRCElement body size was defined based on Frame payload size rather than Datagram payload size! Here's the fix:
```
using CRCElementProps = Protocols::CRCElementProps<
    Driver::Serial::Backend::FrameProps::payload_max_size>;
using DatagramProps = Protocols::DatagramProps<
    Protocols::CRCElementProps<CRCElementProps::payload_max_size>;

using Message = Protocols::Message<
    Application::StateSegment,
    Application::MessageTypeValues,
    DatagramProps::payload_max_size>;
```

Reply:
This PR seems to work fine on the microcontroller.